tests/test_minimal_requirements.txt

```diff
@@ -3,6 +3,7 @@ matplotlib==1.5.3
 sympy==1.0
 nose==1.3.7
 nose-timer==0.7.3
+pytest==5.2.2
```

Comment:
Is this the earliest version we're gonna support? Have you tried using older versions of pytest? If it works with older versions then in this file indicate the oldest version we're supporting, a job will then run on travis to make sure that old version works without error. Having a little flexibility here makes it easier on developers who might get pytest shipped with their OS or in a distribution like anaconda.

Reply:
The last version of pytest that supports python2 is 4.6.x. and starting with 5.0, there's only python3 support. I'll look into 4.6, but from pytest's changelog it seems like there were several bugfixes made to test parameterization. I'm not sure yet if those affect yt.

Reply:
Using pytest 4.6 with python 3 doesn't seem to work, but using it with python 2.7 does. While the tests run, some of the hashes differ between the pytest 4.6 + python 2.7 setup and the pytest 5.2 python 3.7 setup, which is not great.